cdap-cli/src/main/java/co/cask/cdap/cli/command/AbstractSetPreferencesCommand.java

```diff
@@ -91,6 +96,55 @@ protected String determinePattern(String action) {
     throw new RuntimeException("Unrecognized element type: " + type.getShortName());
   }
 
+  protected String determineNewPattern(String action) {
+
+    switch (action) {
+      case "set":
+        return determineNewSetPatternHelper();
+      case "load":
+        return determineNewLoadPatternHelper();
+    }
+    throw new RuntimeException("Unrecognized element type: " + type.getShortName());
+  }
+
+  private String determineNewSetPatternHelper() {
+    switch (type) {
+      case INSTANCE:
+      case NAMESPACE:
+        return String.format("set %s preferences <%s>",
+                             type.getShortName(), ArgumentName.PREFERENCES);
+      case APP:
+      case FLOW:
+      case MAPREDUCE:
+      case WORKFLOW:
+      case SERVICE:
+      case WORKER:
+      case SPARK:
+        return String.format("set %s preferences <%s> <%s>", type.getShortName(), type.getArgumentName(),
+                             ArgumentName.PREFERENCES);
+    }
+    throw new RuntimeException("Unrecognized element type: " + type.getShortName());
+  }
+
+  private String determineNewLoadPatternHelper() {
+    switch (type) {
+      case INSTANCE:
+      case NAMESPACE:
+        return String.format("load %s preferences <%s> <%s>", type.getShortName(),
+                             ArgumentName.LOCAL_FILE_PATH, ArgumentName.CONTENT_TYPE);
+      case APP:
+      case FLOW:
+      case MAPREDUCE:
+      case WORKFLOW:
+      case SERVICE:
+      case WORKER:
+      case SPARK:
+        return String.format("load %s preferences <%s> <%s> <%s>", type.getShortName(), type.getArgumentName(),
+                             ArgumentName.LOCAL_FILE_PATH, ArgumentName.CONTENT_TYPE);
+    }
+    return "None";
```

Comment:
I know this comes from the existing code, but why does this return "None" while the other throws an exception? Seems like this should throw one too?